README.md

```diff
@@ -43,7 +43,7 @@ Run the following command to install the cloud sql proxy operator into
 your kubernetes cluster:
 
 ```shell
-kubectl apply -f https://storage.googleapis.com/cloud-sql-connectors/cloud-sql-proxy-operator/v0.2.0/cloud-sql-proxy-operator.yaml
+kubectl apply -f https://storage.googleapis.com/cloud-sql-connectors/cloud-sql-proxy-operator/v0.3.0/cloud-sql-proxy-operator.yaml
```

Comment:
Is this the only place that this version is listed? Let's try to land these with the release PR next time.

Reply:
There are other places where the version also appears. This is the only one that was missed by the automated version updating script.